### PR TITLE
chore: use require-hashes option

### PR DESCRIPTION
**Pull request by tom-andersen**

### CODE REVIEW

File: .kokoro/requirements.in [hunk 1]
```diff
@@ -0,0 +1,2 @@
+gcp-releasetool
+gcp-docuploader
```
[the remark this answers is not in the document]
Comment (tom-andersen) [reply .kokoro/requirements.in]: Fixed

File: .kokoro/release/publish_javadoc.sh [hunk 2]
```diff
@@ -28,6 +28,7 @@ fi
 pushd $(dirname "$0")/../../
 
 # install docuploader package
+python3 -m pip install --require-hashes -r .kokoro/requirements.txt
 python3 -m pip install gcp-docuploader
```
[the remark this answers is not in the document]
Comment (tom-andersen) [reply .kokoro/release/publish_javadoc.sh]: Fixed

**Comment by mpeddada1** [comment]
@tom-andersen  The versions in the requirements.txt file look great. I'm afraid `owlbot` overwrote some of the changes so we'll need to bring these back (https://github.com/googleapis/java-firestore/pull/1078/commits/1c5e4a99481121674ed03a023bd2bc0ea8b521b1) I've added the `owlbot:ignore` label which should hopefully prevent this from happening again.

**Comment by tom-andersen** [comment]
@mpeddada1 I have reverted the Owl Bot change.

**Review by mpeddada1 — APPROVED** [review]
Thank you so much @tom-andersen! LGTM

**Pull request merged**